data/human/weapons.txt

```diff
@@ -665,8 +665,8 @@ outfit "Meteor Missile Launcher"
 	cost 15000
 	thumbnail "outfit/meteor launcher"
 	"mass" 4
```

Comment:
If capacity and ammo weight isn't changing then mass should come up with outfit space.
```suggestion
	"mass" 8
```